packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -4179,6 +4179,7 @@ class EditableTextState extends State<EditableText>
       case SelectionChangedCause.forcePress:
       case SelectionChangedCause.longPress:
       case SelectionChangedCause.scribble:
+      case SelectionChangedCause.stylusHandwriting:
```

Comment:
I'm actually seeing an error on this line:

```
This case is covered by the previous cases. #unreachable_switch_case
```

Reply:
Oh yes, this error is in the failing analyzer check.

Reply:
Post making my change, I realised this might be needed. Making the change.